src/Service.Tests/CosmosTests/MutationTests.cs

```diff
@@ -797,19 +797,17 @@ public async Task CanPatchItemWithoutVariables()
 }}";
             JsonElement response = await ExecuteGraphQLRequestAsync("patchPlanet", mutation, variables: new());
 
-            JsonElement result = response;
+            Console.WriteLine($"patchPlanet raw response: {response}");
 
-            // In some environments ExecuteGraphQLRequestAsync may return the items array,
-            // so pick the first element when ValueKind is Array.
-            if (response.ValueKind == JsonValueKind.Array && response.GetArrayLength() > 0)
+            if (response.ValueKind == JsonValueKind.Array)
             {
-                result = response[0];
+                Assert.Fail($"Expected patchPlanet to return a single result object, but got an array: {response}");
```

Comment:
Why are we changing the expectation of this test? Why do we expect a single element?